core/include/mm/mobj_ffa.h

```diff
@@ -0,0 +1,75 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2016-2017, Linaro Limited
+ * Copyright (c) 2021, Arm Limited.
+ */
+
+#ifndef __MM_MOBJ_FFA_H
+#define __MM_MOBJ_FFA_H
+
+#include <compiler.h>
+#include <ffa.h>
+#include <mm/core_memprot.h>
+#include <mm/file.h>
+#include <mm/fobj.h>
+#include <mm/mobj.h>
+#include <string_ext.h>
+#include <sys/queue.h>
+#include <tee_api_types.h>
+#include <types_ext.h>
+
+#include <optee_msg.h>
+
+SLIST_HEAD(sp_shared_mem_head_t, sp_mem_access_descr);
```

Comment:
This isn't needed without SPs enabled. We should have the mindset that if SPs aren't enabled it shouldn't have an impact either.